config/test.exs

```diff
@@ -28,3 +28,5 @@ config :logger, level: :warn
 
 # Initialize plugs at runtime for faster test compilation
 config :phoenix, :plug_init_mode, :runtime
+
+config :notesclub, Oban, testing: :inline
```

Comment:
is there any reason using `testing: :inline`?
maybe we can update it to `config : notesclub, Oban, crontab: false, queues: false, plugins: false`.

Reply:
Explicitly setting the testing mode is preferred because it optimizes the config for you automatically. Inline testing mode in particular bypasses the database and runs tests synchronously, as they're inserted. You can switch to `testing: :manual` if you want to make use of testing helpers like `assert_enqueued`. Either way, setting the testing mode is preferred because your configuration is validated in test mode. 

Also, the `crontab` option is deprecated and redundant with `plugins: false`.

Reply:
Thank you, @sorentwo.

Btw, if we use inline testing, is there any need for `queues: false, plugins: false`? If so, is it the preferred way?